acvm_backend_barretenberg/src/acvm_interop/pwg.rs

```diff
@@ -162,3 +165,279 @@ impl PartialWitnessGenerator for Barretenberg {
         Ok(OpcodeResolution::Solved)
     }
 }
+
+// TODO: alter this method so that it only processes one hash per level rather than overriding
+// the one of leaves for each level of the hash path
+fn calculate_merkle_root(
```

Comment:
Why `calculate_merkle_root`? Let's keep it `comptue_merkle_root` like the opcode?

Reply:
This is to match the proposed change to this opcode in https://github.com/noir-lang/acvm/issues/66.

Reply:
Yes ik :) this is a nit, is prefer `compute_merkle_root` like we have it here (https://github.com/noir-lang/acvm/pull/233) so that the names match

Reply:
Ohhh, sorry I misread your comment. Agreed we should match the opcode.